homeassistant/components/mqtt/__init__.py

```diff
@@ -17,18 +17,19 @@
 
 from homeassistant import config_entries
 from homeassistant.components import websocket_api
-from homeassistant.const import (
+from homeassistant.const import (  # noqa: F401
```

Comment:
I don't know what the significance of the noqa is.
Can someone else check?

Reply:
Not sure how I put that in there. I'll remove it.